extensions/amp-script/0.1/amp-script.js

```diff
@@ -649,8 +649,12 @@ export class SanitizerImpl {
             dev().error(TAG, 'Invalid AMP.setState() argument: %s', value);
           });
           if (state) {
-            // Only evaluate updates in case of recent user interaction.
-            const skipEval = !this.userActivationTracker_.isActive();
+            // Only evaluate updates in case of recent user interaction or a small fixed layout.
+            const skipEval =
+              !this.userActivationTracker_.isActive() ||
+              (isLayoutSizeDefined(this.getLayout()) &&
+                this.getLayoutBox().height <= MAX_FREE_MUTATION_HEIGHT);
```

Comment:
Can we dedupe this bit of logic?

Reply:
Yeah, I'm on it. Not the simplest because it needs to be passed through `SanitizerImpl`.